File: 8958 OX퀴즈/8958.c
#include <stdio.h>
#include <string.h>
int main() {
	int testCount;
	scanf("%d", &testCount);
	for (int i = 0; i < testCount; ++i) {
		char inputs[80 + 1];
		scanf("%s", inputs);
		int inputLen = strlen(inputs);
		int accum = 0;
		int answer = 0;

		for (int i = 0; i < inputLen; ++i) {
			if (inputs[i] == 'O') {
				++accum;
				answer += accum;
			}
			else if (inputs[i] == 'X') {
				accum = 0;
			}
		}

		printf("%d\n", answer);
	}

	return 0;
}

File: 8958 OX퀴즈/8958.js
let fs = require('fs');
let inputs = fs.readFileSync('/dev/stdin').toString().split('\n');

let n = parseInt(inputs[0]);
for (let i = 1; i <= n; ++i) {
    let dataStr = inputs[i];
    let dataStrLen = dataStr.length;
    let accum = 0;
    let answer = 0;
    for (let j = 0; j < dataStrLen; ++j) {
        if (dataStr[j] === 'O') {
            ++accum;
            answer += accum;
        }
        else {
            accum = 0;
        }
    }

    console.log(answer);
}